manila/scheduler/manager.py

```diff
@@ -134,6 +134,11 @@ def update_service_capabilities(self, context, service_name=None,
 
     def create_share_instance(self, context, request_spec=None,
                               filter_properties=None):
+        if 'share_instance_properties' in request_spec:
+            LOG.debug(
+                "Attempting to create share instance: "
+                "share_instance_id=%s.",
+                request_spec['share_instance_properties']['id'])
```

Comment:
this log is added when I tried to debug the slow creation issue, which is removed later. is it intended to be added back?

Reply:
I will do a rebase